public/app/features/plugins/admin/types.ts

```diff
@@ -43,6 +43,7 @@ export interface CatalogPlugin {
   isEnterprise: boolean;
   isInstalled: boolean;
   isDisabled: boolean;
+  isPublished: boolean;
```

Comment:
**Idea:** I would probably add a short comment above this line

This is not major at all, more like an idea.
I find that whenever I am uncertain about what an actual property does in a type object, I go to the type definition to see if there is any explanation. This particular property is quite clear, but maybe adding some extra clarification wouldn't hurt. (This would probably be more valuable for less clear fields).

```suggestion
  // `isPublished` is TRUE if the plugin is published to grafana.com 
  isPublished: boolean;
```